bridges/WeLiveSecurityBridge.php

```diff
@@ -3,37 +3,24 @@ class WeLiveSecurityBridge extends FeedExpander {
 
 	const MAINTAINER = 'ORelio';
 	const NAME = 'We Live Security';
-	const URI = 'http://www.welivesecurity.com/';
+	const URI = 'https://www.welivesecurity.com/';
 	const DESCRIPTION = 'Returns the newest articles.';
 
-	private function stripWithDelimiters($string, $start, $end){
-		while(strpos($string, $start) !== false) {
-			$section_to_remove = substr($string, strpos($string, $start));
-			$section_to_remove = substr($section_to_remove, 0, strpos($section_to_remove, $end) + strlen($end));
-			$string = str_replace($section_to_remove, '', $string);
-		}
-
-		return $string;
-	}
-
-
 	protected function parseItem($item){
 		$item = parent::parseItem($item);
 
 		$article_html = getSimpleHTMLDOMCached($item['uri']);
 		if(!$article_html) {
-			$item['content'] .= '<p>Could not request ' . $this->getName() . ': ' . $item['uri'] . '</p>';
+			$item['content'] .= '<p><em>Could not request ' . $this->getName() . ': ' . $item['uri'] . '</p>';
```

Comment:
```returnServerError``` ?

Reply:
When rewriting some bridges, I saw that this behavior was implemented, especially in bridges implementing `FeedExpander`. It seems a good idea as it helps to keep the bridge functional even if requests to page with full contents fails or parsing did not work due to changes on the website.

Let's imagine the following scenario:
 - Site X provides feed F, which is truncated
 - Bridge implements FeedExpander using feed F
 - Site X makes some changes, breaking the ability to request/parse pages
 - With failsave behavior, the bridge fallbacks to just passing fields from feed F
 - With `returnServerError`, the bridge just won't work after changes on site X

So this bridge uses the following approach:
 - Error on article list: `returnServerError`
 - Error on article content: try to keep going

Reply:
I think this is actually a good idea. `$item['content']` already contains the contents from the original feed (notice the `.=`), so there is no harm adding a notice and continue with what we have.

However, you should really close the `em` tag:

```PHP
$item['content'] .= '<p><em>Could not request ' . $this->getName() . ': ' . $item['uri'] . '</em></p>';
```

Reply:
Good catch about the missing `</em>` :open_mouth: Fixing that.